Makefile

```
@@ -18,141 +18,97 @@ IMAGE_LATEST_TAG=$(IMAGE_REGISTRY)/$(REGISTRY_ORG)/$(REGISTRY_REPO):latest
 	- kubectl create -f deploy/operator.yaml
+	make install-service-and-db
+
+.PHONY: uninstall-mss
+uninstall-mss:
+	@echo ....... Create Mobile Security Service Operator and Service in the namespace ${NAMESPACE} ....... 
+	- kubectl delete -f deploy/operator.yaml
+	make uninstall-service-and-db
 
-.PHONY: delete-oper
-delete-oper:
-	@echo Deleting Mobile Security Service Operator and namespace:
+.PHONY: bind-app
+bind-app:
+	@echo ....... Creating an example app CR simulating MDC Bind ......
+	kubectl apply -f deploy/crds/examples/mobile-security-service_v1alpha1_mobilesecurityserviceapp_cr.yaml -n ${APP_NAMESPACE}
+	
+.PHONY: unbind-app
+unbind-app:
+	@echo ....... Deleting an example app CR simulating MDC Bind ......
+	- kubectl delete -f deploy/crds/examples/mobile-security-service_v1alpha1_mobilesecurityserviceapp_cr.yaml -n ${APP_NAMESPACE}
+
+
+.PHONY: teardown
+teardown:
+	make unbind-app
+	@echo ....... Delete Mobile Security Service ....... 
+	make uninstall-mss
+	@echo ....... Delete Operator, Service and namespace ${NAMESPACE} ....... 
+	- kubectl delete -f deploy/service_monitor.yaml
+	- kubectl delete -f deploy/crds/mobile-security-service_v1alpha1_mobilesecurityserviceapp_crd.yaml
 	- kubectl delete -f deploy/crds/mobile-security-service_v1alpha1_mobilesecurityservice_crd.yaml
 	- kubectl delete -f deploy/crds/mobile-security-service_v1alpha1_mobilesecurityservicedb_crd.yaml
-	- kubectl delete -f deploy/crds/mobile-security-service_v1alpha1_mobilesecurityserviceapp_crd.yaml
+	- kubectl delete -f deploy/service_monitor.yaml 
+	- kubectl delete -f deploy/operator_service.yaml 
+	- kubectl delete -f deploy/prometheus-rule.yaml
+	- kubectl delete -f deploy/grafana-dashboard.yaml
 	- kubectl delete -f deploy/cluster_role.yaml
 	- kubectl delete -f deploy/cluster_role_binding.yaml
 	- kubectl delete -f deploy/service_account.yaml
-	- kubectl delete -f deploy/operator.yaml
+	@echo ....... Delete namespace ${NAMESPACE} ....... 
 	- kubectl delete namespace ${NAMESPACE}
 
-.PHONY: create-service-and-db
-create-service-and-db:
-	@echo Creating Mobile Security Service and Database into project:
-	kubectl apply -f deploy/crds/mobile-security-service_v1alpha1_mobilesecurityservice_cr.yaml
-	kubectl apply -f deploy/crds/mobile-security-service_v1alpha1_mobilesecurityservicedb_cr.yaml
-
-.PHONY: create-service-only
-create-service-only:
-	@echo Creating Mobile Security Service App only:
-	kubectl apply -f deploy/crds/mobile-security-service_v1alpha1_mobilesecurityservice_cr.yaml
-
-.PHONY: create-db-only
-create-db-only:
-	@echo Creating Mobile Security Service Database only:
-	kubectl apply -f deploy/crds/mobile-security-service_v1alpha1_mobilesecurityservicedb_cr.yaml
-
-.PHONY: delete-service-and-db
-delete-service-and-db:
-	@echo Deleting Mobile Security Service and Database:
+.PHONY: install-service-and-db
+install-service-and-db:
+	@echo ....... Creating Mobile Security Service and Database into project ....... 
```

IMHO we should not use `into project`. It is a nomenclature very specify of OCP. The appropriate name here would be namespace 
```suggestion
	@echo ....... Creating Mobile Security Service and Database into current namespace  ....... 
```

---

This was a copy and paste, I agree the sentence doesn't sound correct. I will fix it.